magento/pwa-studio
# [doc] update cloud deployment topic

## Description

Updates the Cloud deployment doc with more in-depth steps.

## Related Issue

Closes PWA-1090

## Acceptance

@dpatil-magento 
@tjwiebell 

### Verification Stakeholders

@dpatil-magento 
@tjwiebell 

### Specification



### Verification Steps

1. Navigate to the `pwa-devdocs` directory
1. Run the htlm preview server: `yarn develop`
1. Verify site builds
1. Navigate to `/tutorials/cloud-deploy/`
1. Verify tutorial steps are accurate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
